linz-data-importer/gui/Service_dialog_base.ui

```diff
@@ -791,7 +791,7 @@
                   </font>
                  </property>
                  <property name="text">
-                  <string>* When saving the domain details for the first time the UI may appear to &quot;freeze&quot; while obtaining domain data. Please be patient, this will be cached and updated in the background thereafter.</string>
+                  <string>See "Help" for information on how to get an API key</string>
```

Comment:
I don't understand this change - these messages are saying completely unrelated things.